azurerm/internal/services/servicefabric/resource_arm_service_fabric_cluster.go

```diff
@@ -273,6 +273,123 @@ func resourceArmServiceFabricCluster() *schema.Resource {
 				},
 			},
 
+			"upgrade_description": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"force_restart": {
```

Comment:
we usually put enabled in the name on booleans
```suggestion
						"force_restart_enabled": {
```